src/main/java/de/ait/secondlife/constans/StatusConstans.java

```diff
@@ -0,0 +1,5 @@
+package de.ait.secondlife.constans;
+
+public interface StatusConstans {
```

Comment:
I think that for statuses we will need an Enum, so that we can compare data from the DB for business logic

Reply:
The interface approach does not require enum data conversions.